tests/test_things_example.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import Mock
+
+from examples.things import app, things, ThingsResource
+import falcon
+
+
+def test_app():
+    assert app
+    (resource, method_map, params, uri_template) = app._router.find('/things')
```

Comment:
It would probably be cleaner to do: `resource, *_ = app._router.find('/things')`